mempool/clist_mempool.go

```diff
@@ -418,17 +426,16 @@ func (mem *CListMempool) resCbFirstTime(
 			memTx.addSender(txInfo.SenderID)
 			mem.addTx(memTx)
 			mem.logger.Debug(
-				"added good transaction",
+				"added valid transaction to mempool",
 				"tx", types.Tx(tx).Hash(),
 				"res", r,
 				"height", memTx.height,
 				"total", mem.Size(),
 			)
 			mem.notifyTxsAvailable()
 		} else {
-			// ignore bad transaction
 			mem.logger.Debug(
-				"rejected bad transaction",
+				"rejected invalid transaction",
```

Comment:
Again, why to remove the comment and change the log message?

Reply:
I believe the comment is not needed here. The log message says exactly what is happening in this part of the code.

Reply:
Reverted.